src/modules/Overlay/Ui.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function OverlayModuleUi(step, ui) {
+
+  function setup() {
+    var steps = sequencer.getSteps();
+    steps.forEach(function (_step, index) {
+      if(_step.options && step.options.number === _step.options.number) {
+        if(index === 1){
+          step.ui.querySelector('input[type=range]').value = -1;
```

Comment:
Do we check for `steps.inBrowser` here?

Reply:
We already check that in Module.js before calling this function.